src/coreComponents/linearAlgebra/interfaces/hypre/HypreMGR.cpp

```diff
@@ -56,7 +56,7 @@ void hypre::mgr::createMGR( LinearSolverParameters const & params,
   // Hypre's parameters to use MGR as a preconditioner
   GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetTol( precond.ptr, 0.0 ) );
   GEOS_LAI_CHECK_ERROR( HYPRE_MGRSetMaxIter( precond.ptr, 1 ) );
-  HYPRE_Int logLevel = LvArray::math::min( LvArray::integerConversion< HYPRE_Int >( params.logLevel - 1 ), 0 );
+  HYPRE_Int logLevel = LvArray::math::min( LvArray::integerConversion< HYPRE_Int >( params.logLevel - 1 ), LvArray::integerConversion< HYPRE_Int > ( 0 ) );
```

Comment:
Is this an error?

Reply:
Really gotta be sure about the type of that 0... 

Is there a compiler complaining about this somehow?

Reply:
the mac compiler was complaining about this. It was interpreting 0 as an `int` and `HYPRE_Int` is a `long int` so it says the template parameter type` T` in `LvArray::math::min` is ambiguous. I am guessing other compilers are fine with it though.

Reply:
Interesting failure mode though

Reply:
Hmm I guess most compilers just don't care about the c-type coercion:

![image](https://user-images.githubusercontent.com/4522899/236071256-72e9fe31-9996-47b7-a635-64bf0e8ed434.png)

Reply:
yes, it is `Apple clang version 14.0.0 (clang-1400.0.29.102)` to be precise. I was kind of surprised too but could not think of  a better fix

Reply:
I mean per the standard the conversion is strictly the correct way to handle this, but very surprising in that almost any compiler will make such a literal up-conversion not a warning/error.

Reply:
Sorry for this!

Reply:
blz, nao tem problema nao!